app/views/share/_embed_meta_bookmark.erb

```diff
@@ -0,0 +1,8 @@
+<% if bookmark.bookmarkable.is_a?(Work) %>
+<% if bookmark.tags.exists? %>
+<%= ts("Bookmarker's Tags: ") %><%= bookmark.tags.pluck(:name).join(", ") %><br />
```

Comment:
I'm not sure _why_, but when I try it on webdev, I get `Bookmarker&#39;s Tags:` and `Bookmarker&#39;s Notes:`. I should probably work fine when pasted places, but if we can fix it to show the `'s`, that would be better.